contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -20,12 +22,21 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   bytes4 internal constant CHECK_SELECTOR = KeeperCompatibleInterface.checkUpkeep.selector;
   bytes4 internal constant PERFORM_SELECTOR = KeeperCompatibleInterface.performUpkeep.selector;
   uint256 internal constant PERFORM_GAS_MIN = 2_300;
-  uint256 internal constant CANCELATION_DELAY = 50;
+  uint256 internal constant CANCELLATION_DELAY = 50;
   uint256 internal constant PERFORM_GAS_CUSHION = 5_000;
-  uint256 internal constant REGISTRY_GAS_OVERHEAD = 80_000;
   uint256 internal constant PPB_BASE = 1_000_000_000;
   uint64 internal constant UINT64_MAX = 2**64 - 1;
   uint96 internal constant LINK_TOTAL_SUPPLY = 1e27;
+  // L1_FEE_DATA_PADDING includes 35 bytes for L1 data padding for Optimism
+  bytes public L1_FEE_DATA_PADDING = "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
```

Comment:
I'm wondering if there's a way to instantiate this dynamically by providing a length. e.g. new bytes(35, 0xff) or something?

Reply:
took a search online and could not find a different way to init bytes.